src/types/gqlTypes.ts

```diff
@@ -1130,6 +1130,8 @@ export enum Component {
   SignatureGroup = 'SIGNATURE_GROUP',
   /** SSN input component */
   Ssn = 'SSN',
+  /** Time duration component with hours and minutes */
+  TimeDuration = 'TIME_DURATION',
```

Comment:
The component really only collects *minutes* duration. It doesn't let you select a number of days for example. So lets call it `MINUTES_DURATION`.